### PR TITLE
Updated Readme to reflect all the latest changes and improvements

### PR DESCRIPTION
**Pull request by ader1990**

### CODE REVIEW

**Comment by costingalan** [comment]
I would add:

- [ ] Add a separate part with requirements (ADK version, Host/VM having Hyper-V installed and enabled, VirtIO (specific versions for ws2016 and others)

**Comment by costingalan** [comment]
I would add:

- [ ] The Powershell example script should be written with [splatting workflow](https://technet.microsoft.com/en-us/library/gg675931.aspx)

**Comment by costingalan** [comment]
I would add:

- [ ] Present the users the method to create a Windows image (not syspreped) with the asociated risks.

**Comment by ilenghel** [comment]
Points that would help being mentioned:
What a golden image is, what is its purpose and how can it be generated (step by step guide and tips maybe?)
What is a MaaS image and its purpose (step by step guide and some tips and tricks)
How do you generate a final image that is ready to be used from a golden image (also)
 
A short description on what the script does when building each type of image, following the script execution flow.

File: README.md [hunk 1]
```diff
@@ -1,136 +1,105 @@
...
-You will need pester on your system. It should already be installed on your system if you are running Windows 10. If it is not:
+It should already be installed on your system if you are running Windows 10.<br/>
+If it is not installed you can install it on Windows 10 or greater:
 
 ```powershell
 Install-Package Pester
 ```
 
-Running the actual tests:
+or you can clone it from: https://github.com/pester/Peste://github.com/pester/Pester
```
Comment (costingalan) [inline README.md]: It should be:

`or you can clone it from: https://github.com/Pester/Pester`

File: README.md [hunk 2]
```diff
@@ -1,136 +1,105 @@
...
+## Fast path to create a Windows image
 
-### How to create a Windows template image
+### Requirements:
 
-Requirements:
+* A Windows host, with Hyper-V virtualization enabled and PowerShell >=v4 support
+* A Windows installation ISO or DVD
+* Windows compatible drivers, if required by the target environment
+* Git environment
```
Comment (costingalan) [inline README.md]: The Windows ADK tools are also required.

File: Examples/create-windows-cloud-image.ps1 [hunk 3]
```diff
@@ -0,0 +1,37 @@
+git clone https://github.com/cloudbase/windows-openstack-imaging-tools.git
+pushd windows-openstack-imaging-tools
+Import-Module .\WinImageBuilder.psm1
```
Comment (costingalan) [inline Examples/create-windows-cloud-image.ps1]: In orde to have the updates and MAAS jooks, please add:

`git submodule update --init`

File: Examples/create-windows-online-cloud-image.ps1 [hunk 4]
```diff
@@ -0,0 +1,37 @@
+git clone https://github.com/cloudbase/windows-openstack-imaging-tools.git
+pushd windows-openstack-imaging-tools
+Import-Module .\WinImageBuilder.psm1
```
Comment (costingalan) [inline Examples/create-windows-online-cloud-image.ps1]: In orde to have the updates and MAAS jooks, please add:

`git submodule update --init`

File: Examples/create-windows-online-cloud-image.ps1 [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+# The Windows image file path that will be generated
+$virtualDiskPath = "C:\images\my-windows-image.raw"
+
+# The wim file path is the installation image on the Windows ISO
+$wimFilePath = "D:\Sources\install.wim"
+
+# VirtIO ISO contains all the synthetic drivers for the KVM hypervisor
+$VirtIOISOPath = "C:\images\virtio.iso"
+$virtIODownloadLink = "https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/archive-virtio/virtio-win-0.1.126-2/virtio-win
+.iso"
```
Comment (costingalan) [inline Examples/create-windows-online-cloud-image.ps1]: `.iso` should be on the **13rd** line

File: Examples/create-windows-online-cloud-image.ps1 (hunk 4 above)
Comment (ader1990) [inline Examples/create-windows-online-cloud-image.ps1]: done

File: Examples/create-windows-cloud-image.ps1 (hunk 3 above)
Comment (ader1990) [inline Examples/create-windows-cloud-image.ps1]: done

File: README.md (hunk 2 above)
Comment (ader1990) [inline README.md]: not really, for the windows server versions there is no need for them, but I will add them nonetheless

File: README.md [hunk 6]
```diff
@@ -1,136 +1,113 @@
...
+### Requirements:
 
-* uploading the image to Glance
-* booting an instance on your target hypervisor compute node
-* waiting for the setup to complete (the instance will shutdown once the setup is done) 
-* take a snapshot of the instance which will contain the final sysprepped image ready for your deployments
+* A Windows host, with Hyper-V virtualization enabled, PowerShell >=v4 support<br/>
+and Windows Assessment and Deployment Kit (ADK)
+* A Windows installation ISO or DVD
+i* Windows compatible drivers, if required by the target environment
```
Comment (costingalan) [inline README.md]: with*

**Pull request merged**